src/content/configuration/dev-server.md

```diff
@@ -527,7 +516,7 @@ module.exports = {
 Usage via the CLI
 
 ```bash
-webpack-dev-server --hot-only
+webpack serve --hot only
```

Comment:
`--hot-only` seems correct

Reply:
To be honestly we don't support this yet (regression), I am working on it, but --hot can be boolean or string, so we should supported:
- `webpack serve --hot`
- `webpack serve --no-hot`
- `webpack serve --hot=only` or `webpack serve --hot only`

Reply:
I see, though the option in devServer config object says `hotOnly` - https://webpack.js.org/configuration/dev-server/#devserverhotonly

This means we'll have to change/remove it's flag on `webpack-dev-server`? - https://github.com/webpack/webpack-dev-server/blob/98aa8e1f385b4e5d74ee097fa60362802cda505e/bin/cli-flags.js#L53

Reply:
Yes, we need to to do it